### ISSUE TITLE
Allow Alternate Script Variant for lowercase letter Z that is Even with Code line for Custom Download

### ISSUE THREAD

**Issue by carlomunguia** [MonoLisaFont/feedback#209]
### What feature are you proposing?

The script variant lowercase z is really cool!…but the bottom half of the glyph hangs down below the bottom of any given code line - for me this instantly breaks me out of the flow in reading / coding because of how it sits on the line. 

What I’m proposing is an alternate script variant lowercase z that stays even with the line and could be decided on custom download so people who enjoy the original lowercase script z can continue using it. 

I don’t have a particular design to remedy it, but want to start the conversation to see what the creators think.

**Comment by MarcusSterz** [comment]
Hm, that request points to an interesting problem: you access the script variant through `ss02`. If we place an alternative `z` let’s say on `ss13` (which is the next available free slot) it would not show anyway because `ss02` gets picked up first.
To make that work it would require a second complete set of script variant letters with a different `z` in it. 
@okonet How could we do it otherwise? 

Alternatively we could simply adjust or even eliminate the current script `z`.

**Comment by okonet** [comment]
Yeah that’s an interesting limitation. I’d say let’s keep the z for now as is and explore our options later.

**Comment by carlomunguia** [comment]
Creating a whole other set of variants for just the lowercase letter z isn't realistic - and quite honestly the lowercase z is probably used orders of magnitude lower than many other letters. 

As a programmer I just wanted to make the awareness of how it broke the flow immediately and continued to through the days when that letter Did come up. 

At the same time, it IS a nice glyph you know? I just wish it stayed even on the line! Lol

**Comment by bebraw** [comment]
If the script variant looks good with a regular italic z, one option would be to remove the script z from the current script variant and push it (only current script z) behind a stylistic set of its own as that would allow toggling it separately. That would work around the issue I think.
